<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RVariableInferenceDialogUi</class>
 <widget class="QWidget" name="RVariableInferenceDialogUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Variable Inference</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="fileGroupBox">
     <property name="title">
      <string>File</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>File</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="minimumSize">
         <size>
          <width>200</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="browseButton">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>25</height>
         </size>
        </property>
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="optionGroupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="NormalPlotCheckBox">
        <property name="text">
         <string>Plot the Normal distribution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="LognormalPlotCheckBox">
        <property name="text">
         <string>Plot the Lognormal distribution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ExponentialPlotCheckBox">
        <property name="text">
         <string>Plot the Exponential distribution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="UniformPlotCheckBox">
        <property name="text">
         <string>Plot the Uniform distribution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="LaplacePlotCheckBox">
        <property name="text">
         <string>Plot the Laplace distribution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="RayleighPlotCheckBox">
        <property name="text">
         <string>Plot the Rayleigh distribution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="GammaPlotCheckBox">
        <property name="text">
         <string>Plot the Gamma distribution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ChiSquarePlotCheckBox">
        <property name="text">
         <string>Plot the Chi Square distribution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="LogisticPlotCheckBox">
        <property name="text">
         <string>Plot the Logistic distribution</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="GumbelPlotCheckBox">
        <property name="text">
         <string>Plot the Gumbel distribution</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>OK</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
